app/styles/main.scss

```diff
@@ -590,8 +601,10 @@ btn-mood label::before {
 
 .song-artist {
 	font-size: 15px;
+    display: inline-block;
 	color: rgba(255, 255, 255, 0.6);
 	font-weight: bold;
+    width: 75%;
```

Comment:
Can't this be done using `float: right` or `right: 0`? I have a feeling that this breaks under various resolutions.